components/Dialogs/examples/DialogsTallTextAlertExampleViewController.m

```diff
@@ -93,7 +94,14 @@ - (IBAction)didTapPresent:(id)sender {
     dialogButtonFont = [UIFont fontWithName:urduFontName size:20.0];
   }
   alert.messageFont = dialogBodyFont;
-  alert.buttonFont = dialogButtonFont;
+  for (MDCAlertAction *action in alert.actions) {
+    MDCButton *button = [alert buttonForAction:action];
+    if (button.enableTitleFontForState) {
```

Comment:
Is enableTitleFontForState true by default? Why are there 2 ways to set font for the button?